arangod/Cluster/ClusterComm.h

```diff
@@ -226,7 +226,7 @@ struct ClusterCommResult {
 
   /// @brief return if request was successful, use this on the request
   /// results after a `performRequest`:
-  bool successful() const {
+  bool isGood() const {
```

Comment:
Looks like an unrelated change.
I don't understand its purpose, as the new method name `isGood` is not used/called anywhere else in this PR.
Can you clarify?

Reply:
its not used

Reply:
Exactly. That's why I was wondering what the purpose of the change is.
We can leave it in here, as the code doesn't exist in 3.7 anymore, so the change should not have any relevant effect for 3.6 or future versions, and doesn't need to be ported.